<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortKey</class>
 <widget class="QWidget" name="ShortKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Debug 快捷键：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edDebug"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnDebugSet">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDebugUnSet">
       <property name="text">
        <string>取消设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>代码跳转快捷键：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edToDefine"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnDefineSet">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDefineUnSet">
       <property name="text">
        <string>取消设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
